packages/cli/test/dev/fixtures/go/api/another.go

```diff
@@ -1,10 +1,10 @@
-package handler
+package another
```

Comment:
reverting an earlier change in https://github.com/vercel/vercel/pull/8287/files to make sure this fixture still tests having multiple packages